docs/configuredgraphfactory.adoc

```diff
@@ -67,6 +67,8 @@ You can either use `ConfiguredGraphFactory.create("graphName")`
 or `ConfiguredGraphFactory.open("graphName")`. Learn more about the difference
 between these two options by reading the section below about the `ConfigurationManagementGraph`.
 
+You can also access your graphs by using the bindings. Read more about this in the <<graphandtraversalbindings>> section.
```

Comment:
The name `graphandtraversalbindings` will become either an anchor or a full page, but it's not very readable.

Can you use dashes to separate the words, please? `graph-and-traversal-bindings` is a valid identifier (like `listing-the-graphs` you can see below), and is much more readable and scannable.